xarray/core/formatting.py

```diff
@@ -457,8 +457,8 @@ def short_data_repr(array):
     internal_data = getattr(array, "variable", array)._data
     if isinstance(array, np.ndarray):
         return short_numpy_repr(array)
-    elif hasattr(internal_data, "__array_function__") or isinstance(
-        internal_data, dask_array_type
+    elif hasattr(internal_data, "__array_function__") or is_duck_dask_array(
+        internal_data
```

Comment:
should that be
```suggestion
    elif is_duck_array(internal_data):
```
instead?